bin/main.dart

```diff
@@ -58,58 +87,142 @@ _translateLocalizedFiles(List<String> langCodes, String dirPath) async {
+          (await gtr.translate(sourceString, from: sourceLang, to: targetLang))
+              .text;
+    });
+  });
+}
+
+// Google Translate
+// langStringMap is a map of a language to <key, string>
+// toTranslateMap is a map of <targetLang, sourceLang> to List<key>> - strings to translate
+_translateGoogle(
+    Map<String, Map<String, String>> langStringMap,
+    Map<Tuple2<String, String>, List<String>> toTranslateMap,
+    String projectKey,
+    int numStringsAtOnce) async {
+  await Future.forEach(toTranslateMap.entries, (toTranslate) async {
+    final targetLang = toTranslate.key.item1;
+    final sourceLang = toTranslate.key.item2;
+    final keyList = toTranslate.value;
+    List<String> stringInOutList = List();
+    int num = 0;
+    for (var key1 in keyList) {
+      stringInOutList.add(langStringMap[sourceLang][key1]);
+      ++num;
+      if (num > 0 && num % numStringsAtOnce == 0) {
+        await _translateGoogleString(
+            stringInOutList, sourceLang, targetLang, projectKey);
+        for (int index = 0; index < stringInOutList.length; index++) {
+          langStringMap[targetLang]
+                  [keyList[num - stringInOutList.length + index]] =
+              stringInOutList[index];
+        }
+        stringInOutList.clear();
+      }
     }
+    if (stringInOutList.isNotEmpty) {
+      await _translateGoogleString(
+          stringInOutList, sourceLang, targetLang, projectKey);
+      for (int index = 0; index < stringInOutList.length; index++) {
+        langStringMap[targetLang]
+                [keyList[num - stringInOutList.length + index]] =
+            stringInOutList[index];
+      }
+    }
+  });
+}
+
+// This is where the magic of Google Translate happens
+_translateGoogleString(List<String> stringInOutList, String sourceLang,
+    String targetLang, String projectKey) async {
+  // https://translation.googleapis.com/language/translate/v2?target={YOUR_LANGUAGE}&key=${API_KEY}&q=${TEXT}
+  const baseUrl = 'translation.googleapis.com';
+  const path = 'language/translate/v2';
+  // It's impossible to add several equal query parameters using the 'parameters' map
+  final queryStart =
+      'format=text&target=$targetLang&source=$sourceLang&key=$projectKey';
+  var queryEnd = '';
+  stringInOutList.forEach((text) {
+    queryEnd = queryEnd + '&q=$text';
+  });
+  final url = Uri.https(baseUrl, path).replace(query: queryStart + queryEnd);
+  final data = await http.get(url);
+  if (data.statusCode != 200) {
+    throw http.ClientException('Error ${data.statusCode}: ${data.body}', url);
+  }
+  stringInOutList.clear();
+  final mapList = jsonDecode(data.body)["data"]["translations"];
+  mapList.forEach((map) {
+    stringInOutList.add(map['translatedText']);
   });
-  _updateContent(oldLangStrMap, dirPath);
 }
 
+// Loading of strings from language files
 Future<Map<String, String>> _loadStrings(String lang, String dirPath) async {
   Map<String, String> localizedStrings = Map();
```

Comment:
```suggestion
  final localizedStrings = Map<String, String>();
```

It is not my придирки :) This will be also noticed by Dart lint that it proposed that it can be improved

Reply:
Slightly changed the code, fixed.